### PR TITLE
hardcode to yaml file

### PR DESCRIPTION
**Pull request by chenhonggc**
change `node-problem-detector.yaml` hardcode to yaml file.

### CODE REVIEW

File: docs/tasks/debug-application-cluster/monitor-node-health.md [hunk 1]
```diff
@@ -56,54 +56,16 @@ customized node problems.
...
 
 ***Notice that you should make sure the system log directory is right for your
 OS distro.***
 
 * **Step 2:** Start node problem detector with `kubectl`:
 
 ```shell
-kubectl create -f node-problem-detector.yaml
+$ kubectl create -f node-problem-detector.yaml
 ```
```
Comment (steveperry-53) [inline docs/tasks/debug-application-cluster/monitor-node-health.md]: Our style is not to use the $ in front of commands.

File: docs/tasks/debug-application-cluster/monitor-node-health.md [hunk 2]
```diff
@@ -56,54 +56,16 @@ customized node problems.
 
 * **Step 1:** Create `node-problem-detector.yaml`:
```
Comment (steveperry-53) [inline docs/tasks/debug-application-cluster/monitor-node-health.md]: The pattern we try to follow is this:
Don't have the reader create the yaml file.
Show the yaml file.
In the command, give the path to the yaml file.
Here's an example from another topic:
`kubectl create -f https://k8s.io/docs/tasks/configure-pod-container/memory-request-limit.yaml`

**Comment by chenhonggc** [comment]
@steveperry-53 Thank you for your comment, Please review the PR again, Thanks!

**Comment by chenhonggc** [comment]
@tengqm  please lgtm thanks.

**Pull request merged**